### PR TITLE
preserving folder structure for Saved and Sentry on the Archive mount

### PR DESCRIPTION
**Pull request by prathiraj**
Untested yet, my 3 is out with the family, will test and validate by tonight. Putting it out here for feedback. Only CIFS archive in this change.

### CODE REVIEW

**Comment by prathiraj** [comment]
Test negative, something wrong with the file copy, investigating

File: run/cifs_archive/archive-clips.sh [hunk 1]
```diff
@@ -28,57 +28,58 @@ function connectionmonitor {
 
 function moveclips() {
   ROOT="$1"
-  PATTERN="$2"
+  SUB="$2"
```
Comment (marcone) [inline run/cifs_archive/archive-clips.sh]: You could keep this a 2-argument function and just add `SUB=$(dirname "$ROOT")` here.
That way a lot of the code below can stay the same.

File: run/cifs_archive/archive-clips.sh [hunk 2]
```diff
@@ -28,57 +28,58 @@ function connectionmonitor {
...
-        rm "$ROOT/$file_name"
+        log "'$SUB/$file_name' is only $size bytes"
+        rm "$ROOT/$SUB/$file_name"
         NUM_FILES_DELETED=$((NUM_FILES_DELETED + 1))
       else
-        log "Moving '$file_name'"
+        log "Moving '$SUB/$file_name'"
         outdir=$(dirname "$file_name")
-        if mv -f "$ROOT/$file_name" "$ARCHIVE_MOUNT/$outdir"
+        if mv -f "$ROOT/$SUB/$file_name" "$ARCHIVE_MOUNT/$outdir"
```
Comment (marcone) [inline run/cifs_archive/archive-clips.sh]: I think this should be `mv -f "$ROOT/$SUB/$file_name" "$ARCHIVE_MOUNT/$SUB/$outdir"` (or `mv -f "$ROOT/$file_name" "$ARCHIVE_MOUNT/$SUB/$outdir` if keeping this a 2-arg function like I said above.

File: run/cifs_archive/archive-clips.sh (hunk 1 above)
Comment (marcone) [reply run/cifs_archive/archive-clips.sh]: (actually `SUB=$(basename $(dirname $ROOT))`)

File: run/cifs_archive/archive-clips.sh (hunk 2 above)
Comment (prathiraj) [reply run/cifs_archive/archive-clips.sh]: figure this out in test, fixed and testing again

File: run/cifs_archive/archive-clips.sh (hunk 1 above)
Comment (prathiraj) [reply run/cifs_archive/archive-clips.sh]: got it, makes it much simpler, thanks.

**Comment by marcone** [comment]
Can you squash these commits together and reupload as a single commit?

**Pull request closed without merging**